hbt/config/configs_hbt.py

```diff
@@ -43,6 +44,10 @@ def add_config(
     assert run in {2, 3}
     assert year in {2016, 2017, 2018, 2022, 2023}
 
+    # renamings
+    sync_requested = sync
+    del sync
+
```

Comment:
I don't like that we rename the variable right after declaring it. I find that `sync` as flag is clear enough. 
```suggestion
```

Reply:
Everything else looks good to me. Its nice to see that we now have a lazy_loader for configs.

Reply:
"now" = July 15 ;)

Reply:
Also, agreed. I renamed it to `sync_mode` as `sync` itself can be read as a verb / instruction.